CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(CMaizeSuperbuild VERSION 1.0.0)
+
+include(ExternalProject)
+
+include(GNUInstallDirs)
+
+ExternalProject_Add(
+    "${PROJECT_NAME}_core"
+    SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/src"
+    CMAKE_ARGS -DCMAKE_CXX_COMPILER="${CMAKE_CXX_COMPILER}"
+               -DCMAKE_CXX_STANDARD="${CMAKE_CXX_STANDARD}"
+               -DCMAKE_CXX_EXTENSIONS="${CMAKE_CXX_EXTENSIONS}"
```

Comment:
Do you really need to set this? I feel like CMake takes care of this automatically.

Reply:
All the super build models I have seen use this structure, but I can look into this more.

Reply:
I'm referring to setting `CMAKE_CXX_EXTENSIONS`. I don't think you need to mess with it.

Reply:
Sounds good, should me removed now.